spring-rabbit-junit/src/main/java/org/springframework/amqp/rabbit/junit/LogLevels.java

```diff
@@ -57,4 +58,11 @@
 	 */
 	String level() default "DEBUG";
```

Comment:
You know, I've checked a `Level.toLevel()` and it defaults to `DEBUG`. For now... It might be changed eventually.
So, let's revert this change to the empty string and really rely on the default in the target Log4J impl!